src/App.vue

```diff
@@ -38,13 +39,8 @@
 </template>
 
 <script>
-import FormCheckbox from './components/FormCheckbox';
```

Comment:
ked si ro prerobil na Plugin treba upravit aj README

Reply:
btw form-item ma plugin samostane v priecinku /plugin - uprav to nech maju nase komponenty rovnaku strukturu
![image](https://user-images.githubusercontent.com/25453097/74824497-8be36200-5308-11ea-874b-540e5e7db5ce.png)

Reply:
Done

Reply:
Done